infrastructure/alerts.tf

```diff
@@ -48,3 +48,39 @@ module "fpl-grant-case-access-failure-alert" {
   resourcegroup_name         = "${local.alert_resource_group_name}"
   enabled                    = "${var.enable_alerts}"
 }
+
+module "fpl-exceptions-alert" {
+  source                     = "git@github.com:hmcts/cnp-module-metric-alert"
+  location                   = "${var.appinsights_location}"
+  app_insights_name          = "${var.product}-${var.component}-appinsights-${var.env}"
+  alert_name                 = "${var.product}-exceptions-alert"
+  alert_desc                 = "All exceptions within FPL"
+  app_insights_query         = "exceptions | where operation_Name !contains "health" | project timestamp, severityLevel, operation_Name, outerMessage, ['details'] | sort by severityLevel"
+  custom_email_subject       = "Alert: FPL all exceptions"
+  frequency_in_minutes       = 5
+  time_window_in_minutes     = 5
+  severity_level             = "3"
+  action_group_name          = "${var.product}-support"
+  trigger_threshold_operator = "GreaterThan"
+  trigger_threshold          = 0
+  resourcegroup_name         = "${local.alert_resource_group_name}"
+  enabled                    = "${var.enable_alerts}"
+}
+
+module "fpl-health-failure-alert" {
+  source                     = "git@github.com:hmcts/cnp-module-metric-alert"
+  location                   = "${var.appinsights_location}"
+  app_insights_name          = "${var.product}-${var.component}-appinsights-${var.env}"
+  alert_name                 = "${var.product}-health-failure-alert"
+  alert_desc                 = "Failed health requests"
+  app_insights_query         = "requests | where url contains "health" | where resultCode != "200" | project timestamp, resultCode, duration"
```

Comment:
Duration might not be super helpful (health is always quick), potentially operationId so we can jump in immediately :).

Thanks